AWS_CloudWatch/FIleMetricPublisher.py

```diff
@@ -0,0 +1,79 @@
+from Functions.MetricPublisher import MetricPublisher
+from Functions.Metrics import CounterMetric
+import unittest
+from datetime import datetime
+import json
+import os
+import time
+
+
+class FileMetricPublisher(MetricPublisher):
+    def __init__(self,file_location:str,name_space:str):
+        self.file_location = file_location
+        self.name_space = name_space
+        self.counter_metric = CounterMetric(name_space=self.name_space)
```

Comment:
why?

the publisher and the metric are different objects - the publisher should have a method to publish instead